dockerfile-configs/common-components.yaml

```
@@ -126,6 +126,13 @@
       kubectl krew install get-all
     info: |
       ketall is to enable `kubectl get-all -n namespace_name`, for listing all resources in some namespace
+  - name:  crictl
+    command: |
+      cd /root/dotfiles/;\
+      chmod +x install_crictl.sh;\
+      ./install_crictl.sh
+    info: |
+      crictl is cli tools for Kubelet Container Runtime Interface (CRI).
```

You can put this this above where the curl commands are and remove the `install_crictl.sh` file

```suggestion
curl:
  - name:  crictl
    version: v1.17.0
    from: https://github.com/kubernetes-sigs/cri-tools/releases/download/{version}/crictl-{version}-linux-amd64.tar.gz
    to: /crictl.tar.gz
    command: |
      tar -xf /crictl.tar.gz -C /usr/local/bin;\
      rm -f /crictl.tar.gz;\
      echo "runtime-endpoint: unix:///run/containerd/containerd.sock" > /etc/crictl.yaml;\
      echo "image-endpoint: unix:///run/containerd/containerd.sock" >> /etc/crictl.yaml;\
      echo "timeout: 10" >> /etc/crictl.yaml
    info: "crictl is a cli tool for Kubelet Container Runtime Interface (CRI). The current configuration is: image-endpoint: unix:///hostroot/run/containerd/containerd.sock and runtime-endpoint: unix:///hostroot/run/containerd/containerd.sock. You can find all .sock files under /hostroot where the root directory of the host is mounted by default.
```